modules/cockroachdb/options.go

```diff
@@ -67,3 +68,23 @@ func WithTLS(cfg *TLSConfig) Option {
 		o.TLS = cfg
 	}
 }
+
+// ClusterDefaults are the settings recommended by Cockroach Labs for testing clusters.
+// See https://www.cockroachlabs.com/docs/stable/local-testing for more information.
+var ClusterDefaults []string = []string{
```

Comment:
suggestion: you don't need to type definition
```suggestion
var ClusterDefaults = []string{
```